### PR TITLE
fix trio.Event docstring

### PR DESCRIPTION
**Pull request by belm0**

### CODE REVIEW

File: trio/_sync.py [hunk 1]
```diff
@@ -23,18 +23,18 @@ class Event:
...
     inspired by :class:`threading.Event`.
 
-    An event object with an internal boolean flag, representing whether
+    An event object has an internal boolean flag, representing whether
     the event has happened yet. The flag is initially False, and the
-    :meth:wait method waits until the flag is True. If the flag is
-    already True, then :meth:wait returns immediately. (If the event has
+    :meth:`wait` method waits until the flag is True. If the flag is
+    already True, then :meth:`wait` returns immediately. (If the event has
     already happened, there's nothing to wait for.) The :meth:set method
```
Comment (njsmith) [inline trio/_sync.py]: missed one here :-)

File: trio/_sync.py [hunk 2]
```diff
@@ -23,18 +23,18 @@ class Event:
...
     sets the flag to True, and wakes up any waiters.
 
     This behavior is useful because it helps avoid race conditions and
     lost wakeups: it doesn't matter whether :meth:set gets called just
```
Comment (njsmith) [inline trio/_sync.py]: and another

Comment (belm0) [reply trio/_sync.py]: thank's for catching these!

**Comment by njsmith** [comment]
looks good to me! feel free to merge once CI psases

**Pull request merged**